publication/src/main/resources/reports-metadata.txt

```diff
@@ -100,7 +102,9 @@ N_A4,National Aggregate,A4,,"Disposition of preapprovals for conventional home-p
 N_B,National Aggregate,B,,"Loan pricing information for conventional loans by incidence and level"
 N32,National Aggregate,3-2,,"Pricing Information for First and Junior Lien Loans Sold by Type of Purchaser (includes originations only)"
 N41,National Aggregate,4-1,,"Disposition of applications for FHA, FSA/RHS, and VA home-purchase loans, 1- to 4- family and manufactured home dwellings, by race, ethnicity, gender and income of applicant"
+N42,National Aggregate,4-2,,"Disposition of applications for conventional home-purchase loans 1- to 4- family and manufactured home dwellings, by race, ethnicity, gender and income of applican"
```

Comment:
Missed one letter at the end 😄

Reply:
woooo copypasta 😅 . Fixed in e22a836acd1550f881e3fc5e8544ca3acd483c7a.